src/components/Button/Base.js

```diff
@@ -1,9 +1,18 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { ThemeProvider } from "styled-components";
 
 import { StyledButton, StyledButtonLink } from "./Base.styles";
 import { BUTTON_VARIANTS, BUTTON_SIZES } from "./constants";
 import getRelByTarget from "../../utils/link";
+import { themes } from "../../theme";
+
+/**
+ * Provides a default `global` theme when no theme provider is used higher up
```

Comment:
I feel like an error is an acceptable outcome when no `ThemeProvider` or an incorrect `themeName` is used higher up in the components tree... if the error is uninformative, perhaps validating for the `theme` `propType` and adding a more informative error could be another approach.